### ISSUE TITLE
wjelement supports json pointer?

### ISSUE THREAD

**Issue by wrenashe** [petehug/wjelement-cpp#6]
wjelement supports json pointer?

**Comment by petehug** [comment]
Yes it does, but rapidjson (https://rapidjson.org/) is a much superior library and thus I stopped maintaining this one.

**Comment by wrenashe** [comment]
> superior

Thanks, Pete.

Also wonder if the schema $ref definition below is supported,
 "$ref":"Account.schema.json#/definitions/Accounts/1/properties/link"
